docs/project/pull_request_workflow.md

```diff
@@ -101,29 +102,28 @@ review you typically want to leave the development history undisturbed until the
 end so that comments on any particular increment aren't lost. We typically use
 the GitHub squash-and-merge functionality to land things.
 
-### Managing pull requests with multiple commits
```

Comment:
@chandlerc Though these recommendations are still valid, this doesn't really apply anymore. Are you ok with removing this section, or do you prefer to keep it?

Reply:
I'd be fine removing this, I don't think we've ever done what's described here -- we pretty exclusively squash-and-merge, not rebase.